content/docs/tasks/telemetry/distributed-tracing/overview/index.md

```diff
@@ -81,8 +81,8 @@ When you make downstream calls in your applications, make sure to include these
 
 ## Trace sampling
 
-Istio captures a trace for all requests by default. For example, when
-using the Bookinfo sample application above, every time you access
+When installing with demo profile, Istio captures a trace for all requests by default.
```

Comment:
`Istio captures a trace for all requests by default when installing with the demo profile.`

Reply:
Done.